src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.builds

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <!-- This property must be set to the same value as $(PackageOutputPath) for the nuspecs and nupkgs to be binplaced to the intended location. -->
+    <OutputPath>$(PackageOutputPath)</OutputPath>
+  </PropertyGroup>
+
+  <!-- We always build the identity/redirection package. However, the platform specific runtime-*.nupkg is built based upon the target OS we are building the product for. -->
+  <ItemGroup>
+    <Project Include="Microsoft.NETCore.Native.pkgproj">
```

Comment:
> runtime.native.XXX tries to follow that approach, unnecessarily

Naming tends to be imperfect, but it is very valuable to be consistently imperfect. 

> And it's similar to the other packages in CoreCLR repo, which are structurally similar to this one, such as Microsoft.NETCore.JIT.

I do not think that this package is structurally similar to JIT.